datafusion/physical-expr/src/aggregate/count_distinct/strings.rs

```diff
@@ -92,403 +86,3 @@ impl<O: OffsetSizeTrait> Accumulator for StringDistinctCountAccumulator<O> {
         std::mem::size_of_val(self) + self.0.size()
     }
 }
-
-/// Maximum size of a string that can be inlined in the hash table
```

Comment:
This code / algorithm still exists, but it has been moved into `string_map.rs` and made general